FirebaseCore/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ if(NOT APPLE)
   return()
 endif()
 
+# GoogleUtilities is an external dependency without its own cmake configuration.
+include(GoogleUtilities)
```

Comment:
This doesn't seem like it belongs here; GoogleUtilities isn't isolated to just FirebaseCore, is it?

A better place would be to put this before line 265 of the top-level CMakeLists.txt.

Reply:
Putting it there causes the duplicate symbol issues I was seeing before 84ccde5

See https://github.com/firebase/firebase-ios-sdk/pull/7324/checks?check_run_id=1730495590#step:6:6581

Is there a way to link GoogleUtilities only once instead of into all of the intermediate libraries if it's in the top level CMakeLists.txt?

Reply:
Internal bug to investigate moving the include back to the top-level CMakeLists.txt at b/178027829

Reply:
Let's set this issue aside for the moment to unblock the GoogleUtilities move.